integration/bundle/init_default_python_test.go

```diff
@@ -58,7 +58,10 @@ func testDefaultPython(t *testing.T, pythonVersion string) {
 	require.NoError(t, err)
 	require.NotNil(t, user)
 	testdiff.PrepareReplacementsUser(t, replacements, *user)
-	testdiff.PrepareReplacements(t, replacements, wt.W)
+	testdiff.PrepareReplacementsWorkspaceClient(t, replacements, wt.W)
+	testdiff.PrepareReplacementsUUID(t, replacements)
+	testdiff.PrepareReplacementsNumber(t, replacements)
+	testdiff.PrepareReplacementsTemporaryDirectory(t, replacements)
```

Comment:
This group of calls belong in testdiff as well? so we get the same set of replacements across all tests? (unless author needs to pick and choose).

testdiff.PrepareReplacementsAll(t, replacements, user, wt.W).

Reply:
I'd rather pick and choose for now. For the acceptance tests, for example, I specifically don't want to include the replacement of numbers.